coprocessor/fhevm-engine/tfhe-worker/src/tfhe_worker.rs

```diff
@@ -303,9 +318,9 @@ async fn query_for_work<'a>(
 WITH selected_computations AS (
   (
     SELECT DISTINCT
-      c_creation_order.transaction_id
+      c_creation_order.transaction_id, c_creation_order.block_hash
```

Comment:
should there be a condition that check if this block was reorg?
shouldn't the host listener add this information to the computation table?

It will allow to ensure those computation are not picked up